met-web/src/components/survey/submit/ActionContext.tsx

```diff
@@ -55,7 +55,11 @@ export const ActionProvider = ({ children }: { children: JSX.Element }) => {
                     text: 'Survey was successfully submitted',
                 }),
             );
-            navigate(`/engagement/view/${savedSurvey.engagement.id}`);
+            navigate(`/engagement/view/${savedSurvey.engagement.id}`, {
+                state: {
```

Comment:
cool way of passing the state, i like it

Reply:
thanks! :)